styles/grids.css

```diff
@@ -0,0 +1,19 @@
+.grid--two {
+  display: grid;
+  grid-template-columns: auto auto;
+  width: 100%;
+  text-align: center;
+}
+
+.grid--three {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  width: 100%;
+  text-align: center;
+}
+
+.box {
+  border: 1px solid;
```

Comment:
Would suggest to use rem for all, instead of px.

Reply:
Rest is good to merge.

Reply:
> Would suggest to use rem for all, instead of px.

Reuben we can indeed use px for sizes lesser than 10px since they probably won't make any difference and it is harder to understand 0.0625rem such as in this case :)